docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1.4
```

Comment:
What is the purpose of this new file please?
Is it just to test a new image that has been built?

Reply:
Buildkit needs a Dockerfile to build the image from. It is also easier to see what is inside the image.